commands.go

```diff
@@ -127,7 +127,7 @@ type dropMeasurementCommand struct {
 type createMeasurementSubcommand struct {
 	Name          string              `json:"name"`
 	Tags          []map[string]string `json:"tags"`
-	marshaledTags []string            // local cache...don't marshal
+	marshaledTags map[string]struct{} // local cache...don't marshal
```

Comment:
Style suggestion -- pull this field into its own section of the struct definition, separated by a newline from the exported fields.